packages/shopping/src/services/enforcer.ts

```diff
@@ -2,7 +2,10 @@ import * as casbin from 'casbin';
 import * as path from 'path';
 
 export async function createEnforcer() {
-  const conf = path.resolve(__dirname, '../fixtures/casbin/rbac_model.conf');
-  const policy = path.resolve(__dirname, '../fixtures/casbin/rbac_policy.csv');
-  return await casbin.newEnforcer(conf, policy);
+  const conf = path.resolve(__dirname, '../../fixtures/casbin/rbac_model.conf');
+  const policy = path.resolve(
+    __dirname,
+    '../../fixtures/casbin/rbac_policy.csv',
+  );
+  return casbin.newEnforcer(conf, policy);
```

Comment:
good catch, and we can turn `createEnforcer` into a sync function since `casbin.newEnforcer()` is sync.

Reply:
casbin.newEnforcer returns a promise, so to resolve it createEnforcer has to be async. The reason I removed the await is because of lint errors and await is added in a return from an async function internally.